CondCore/CondDB/src/IOVProxy.cc

```diff
@@ -293,8 +293,38 @@ namespace cond {
         throwException("The transaction is not active.", ctx);
     }
 
+    bool printIOVSequenceDiagnostics(const IOVProxyData& iovProxyData,
```

Comment:
This function seems to return `true` unconditionally, and the caller seems to ignore the return value. How about not returning anything?
```suggestion
    void printIOVSequenceDiagnostics(const IOVProxyData& iovProxyData,
```